# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/nsBaseWidget.cpp
# Commit: 7398bc391379
# Full Hash: 7398bc391379a76e4bdde7dd13b0410ea7137ad7
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2021-08-18 15:53:06
# Regressor Bug: 1726063
# File Overlap Count: 2
# Description:
#   Bug 1726063 - Remove option to use BasicLayerManager. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D122796
# ==============================================================================

diff -r f62cad8b2ebd -r 7398bc391379 widget/nsBaseWidget.cpp
--- a/widget/nsBaseWidget.cpp	Tue Aug 17 21:35:35 2021 +0000
+++ b/widget/nsBaseWidget.cpp	Tue Aug 17 23:19:44 2021 +0000
@@ -1533,11 +1533,7 @@
 }
 
 WindowRenderer* nsBaseWidget::CreateBasicLayerManager() {
-  if (StaticPrefs::gfx_basic_layer_manager_force_enabled()) {
-    return new BasicLayerManager(this);
-  } else {
-    return new FallbackRenderer;
-  }
+  return new FallbackRenderer;
 }
 
 CompositorBridgeChild* nsBaseWidget::GetRemoteRenderer() {
